src/components/text-editor/prosemirror-adapter/menu/menu-command-utils/list-utils.ts

```
@@ -0,0 +1,243 @@
+    );
+};
+
+const convertListAttributes = (
+    fromType: NodeType,
+    toType: NodeType,
+    attrs: Record<string, any>,
+) => {
+    const newAttrs = { ...attrs };
+    if (fromOrderedToBulletList(fromType, toType)) {
+        // Bullet lists generally do not need an "order" attribute.
+        delete newAttrs.order;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)